### DISCUSSION TITLE
How do I install gap_si_surface

### DISCUSSION THREAD

**Discussion by makinzm** [libAtoms/QUIP#403] [Q&A]
I'm going to study GAP using this tutorial : https://libatoms.github.io/GAP/gap_si_surface.html?highlight=gap_si_surface

However, I cannot understand how to install gap_si_surface.

# What I did

My environment is MacOS and I use pip.

```
pip3 install ase
pip3 install quippy-ase
```

And then I searched this site(https://pypi.org/search/?q=gap_si_surface) for gap_si_surface, but I cannot find this package.

I would appreciate it , if you could teach me.

addition)

I wonder where ``dftb-params.xml`` is. Therefore, I would appreciate it too.

**Comment by jameskermode** [comment]
The package is included in the GAP repo documentation: https://github.com/libAtoms/GAP/tree/main/doc_src/gap_si_surface

**Reply by makinzm** [reply]
Thank you for teaching me.
